src/lib/proof_system.ts

```diff
@@ -925,6 +945,7 @@ ZkProgram.Proof = function <
     static tag = () => program;
   };
 };
+ExperimentalZkProgram.Proof = ZkProgram.Proof;
```

Comment:
this was a bug which accidentally broke `Experimental.ZkProgram`, fixed it while I was here